### PR TITLE
blog: announce the new blog site

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Once reviewed/merged I'll port it to the old site as well, which should be the last blog there. That will just leave the redirects and niggles for later today/this week.

### CODE REVIEW

File: _posts/2015-11-04-presenting_the_new_foreman_blog.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+As part of my role as the new Community Lead, I'll definitely be posting more
+about all sorts of Foreman related things, as well as my own activities to keep
+the community informed. In addition, the main goal for changing to Markdown was
+to enable community contribution, so ...
+
+If you see any bugs or problems with any old blog posts, please do let us know
+in the [issue tracker](https://github.com/theforeman/theforeman.org/issues) or
+even better, open a Pull Request to fix the problem (see
+[Contributing](http://theforeman.org/contribute.html#Documentationandwebsite)
```
Comment (domcleal) [inline _posts/2015-11-04-presenting_the_new_foreman_blog.md]: `{{site.baseurl}}`

File: _posts/2015-11-04-presenting_the_new_foreman_blog.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## What's new?
+
+In terms of content - very little. The old posts have been imported from
+Blogger via their XML export tool, and then converted to Markdown via the
+awesome [Pandoc](pandoc.org) tool. Images were downloaded from blogger too, and
+Gists loaded via the *jekyll-gist* gem which was already part of the website
+stack. However, the vast majority of the text is unchanged.
+
+The URL has changed though - it's now
+[http://theforeman.org/blog](http://theforeman.org/blog)
```
Comment (domcleal) [inline _posts/2015-11-04-presenting_the_new_foreman_blog.md]: ditto, and feed.xml

File: _posts/2015-11-04-presenting_the_new_foreman_blog.md (hunk 1 above)
Comment (GregSutcliffe) [inline _posts/2015-11-04-presenting_the_new_foreman_blog.md]: Fixed.

File: _posts/2015-11-04-presenting_the_new_foreman_blog.md (hunk 2 above)
Comment (GregSutcliffe) [inline _posts/2015-11-04-presenting_the_new_foreman_blog.md]: Fixed, thanks.

File: _posts/2015-11-04-presenting_the_new_foreman_blog.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+## What's new?
+
+In terms of content - very little. The old posts have been imported from
+Blogger via their XML export tool, and then converted to Markdown via the
+awesome [Pandoc](pandoc.org) tool. Images were downloaded from blogger too, and
+Gists loaded via the *jekyll-gist* gem which was already part of the website
+stack. However, the vast majority of the text is unchanged.
+
+The URL has changed though - it's now
+[http://theforeman.org/blog]({{site.baseurl}}/blog)
```
Comment (domcleal) [inline _posts/2015-11-04-presenting_the_new_foreman_blog.md]: Remove the extra `/`, it'll break the link.

File: _posts/2015-11-04-presenting_the_new_foreman_blog.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+As part of my role as the new Community Lead, I'll definitely be posting more
+about all sorts of Foreman related things, as well as my own activities to keep
+the community informed. In addition, the main goal for changing to Markdown was
+to enable community contribution, so ...
+
+If you see any bugs or problems with any old blog posts, please do let us know
+in the [issue tracker](https://github.com/theforeman/theforeman.org/issues) or
+even better, open a Pull Request to fix the problem (see
+[Contributing]({{site.baseurl}}/contribute.html#Documentationandwebsite)
```
Comment (domcleal) [inline _posts/2015-11-04-presenting_the_new_foreman_blog.md]: ditto, extra `/`

**Comment by lzap** [comment]
Side note: Links in our feed have extra slash in them. It's worth fixing _before_ we announce the blog (the fix might duplicate all the records in user's readers).

Example link: http://theforeman.org//2015/10/foreman-community-newsletter-october.html

**Comment by domcleal** [comment]
I think duplication is inevitable as the IDs will have changed domain, but sure, now is a good time to fix it.

**Comment by GregSutcliffe** [comment]
Thanks for the typos.

Re the double slashes - jekyll-feed is doing that (here: https://github.com/jekyll/jekyll-feed/blob/98cd38e7220b7e692f7f0f1186228a5673267adf/lib/feed.xml#L3). I don't see a way to configure it, other than to drop `baseurl`, but we use that extensively in the manual... perhaps we could rename it to a custom variable for the manual?

**Comment by domcleal** [comment]
I think it has a particular meaning in Jekyll, not just a custom-named variable.  It's set or used when you run the server on a non-root URL too, e.g. on GitHub.  I'd be very wary about messing with it.

**Comment by GregSutcliffe** [comment]
Fair enough. In that case I don't see that we have much option but to leave the double slash in there then :(

**Comment by GregSutcliffe** [comment]
Interestingly the [default Jekyll config](http://jekyllrb.com/docs/configuration/#default-configuration) seems to use `baseurl: ""`. I tried it locally, and as @domcleal says, it broke a lot of things, however, [this commit](https://github.com/GregSutcliffe/theforeman.org/commit/0e801bec4b8cd6b3aba0114db7b89662c4b075c0) seems to resolve all the issues I could find. Thoughts?

**Comment by GregSutcliffe** [comment]
Scratch that, I was over-complicating as usual. [This page](http://jekyllrb.com/docs/github-pages/#project-page-url-structure) points out using "{{site.baseurl}}/" for includes, which seem to be the only broken thing. See [this commit](https://github.com/GregSutcliffe/theforeman.org/commit/b93092f53627a0528c4dc5d532369b24e46a45b3) for my attempt.

EDIT: the point being, that `baseurl` shouldn't ever have a trailing slash - it's "" or "/suburl"

**Comment by domcleal** [comment]
Yes, I think you're right about the trailing slash.  It seems quite safe to remove because we have the `<base>` element.  Both were added in #17.

Suggest you make your commit into a PR?

**Comment by GregSutcliffe** [comment]
@domcleal cool, PR #471 submitted. I assume we'll depend on that here, to get the feed right.

**Comment by GregSutcliffe** [comment]
Date updated since we didn't get this out yesterday. I'd _really_ like to get this out today if we can, so I can check if the G+ comments work on new posts.

**Comment by domcleal** [comment]
Thanks, both merged.

**Pull request merged**